marklogic-data-hub-central/ui/src/config/tooltips.config.tsx

```diff
@@ -182,11 +182,11 @@ const ModelingTooltips = {
   relationshipNameInfo: function (entityName) {
     return (
       <span>
-        The name that identifies the relationship between the source and target entity types. The relationship is saved
-        as a entity property in the <strong>{entityName}</strong> entity type.
+        The name that identifies the relationship between the source and target concept. The relationship is saved as a
```

Comment:
The text here should be "The relationship is saved as **an** entity property..."

Reply:
Fixed!